packages/mobile/src/components/comments/CommentText.tsx

```diff
@@ -37,21 +37,18 @@ const TimestampLink = (props: TimestampLinkProps) => {
   const dispatch = useDispatch()
   const { track } = useCurrentCommentSection()
   const lineup = useSelector(getLineup)
-  const { track_id: trackId, stream_conditions: streamConditions } = track
+  const { track_id: trackId } = track
 
-  const isUSDCPurchaseGated = isContentUSDCPurchaseGated(streamConditions)
   const { hasStreamAccess } = useGatedContentAccess(track)
```

Comment:
learned recently that the track should have an access key on it that you can check for steam access, so no more need for this hook apparently